onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_support_checker.cc

```diff
@@ -1260,15 +1262,25 @@ class GemmOpSupportChecker : public BaseOpSupportChecker {
       const InitializedTensorSet& /* initializers */, const NodeUnit& node_unit,
       const OpSupportCheckParams& /* params */) const override;
   int GetMinSupportedOpSet(const NodeUnit& node_unit) const override;
+
+  bool IsNodeUnitTypeSupported(const NodeUnit& /* node_unit */) const override { return true; }
```

Comment:
Do you also want to support QDQMatmul?
Because when a QDQMatmul is passed to this, it will return true

Same below